### PR TITLE
Building on overkill with configs

### PR DESCRIPTION
**Pull request by mike-robertson**
The goal of this pull request is to build on the overkill code created by @EnDscx by putting it behind a toggle. I may also add additional configurations to allow offset positioning of the overkill text.

### CODE REVIEW

**Review by Justw8 — COMMENTED** [review]
I always liked the idea of this, but never wanted to commit to adding it, but having it as optional thing - so by default off; btw. I can be okay with it. In the end this addon was made to have the events attached to nameplates; so this sorta diverges from that, but I understand the need and it being disapointing that you cannot see your big final hits.

File: NameplateSCT.lua [hunk 1]
```diff
@@ -993,13 +1064,24 @@ local menu = {
...
         },
 
+        displayOverkill = {
+            type = 'toggle',
+            name = L["Display Overkill"],
+            desc = L["Display your overkill for a target over your own nameplate"],
+            get = function() return NameplateSCT.db.global.shouldDisplayOverkill; end,
+            set = function(_, newValue) NameplateSCT.db.global.shouldDisplayOverkill = newValue; end,
+            order = 6,
+            disabled = function() return not NameplateSCT.db.global.shouldDisplayOverkill; end;
```
[the remark this answers is not in the document]
Comment (mike-robertson) [reply NameplateSCT.lua:1074]: Ah, yes I think you're right. I duplicated one of the other configs when I was setting this up and didn't read this too closely. Pushing up a fix now.

**Comment by mike-robertson** [comment]
> The goal of this pull request is to build on the overkill code created by @EnDscx by putting it behind a toggle. I may also add additional configurations to allow offset positioning of the overkill text.

Sweet. I set the default to disabled as well. Since you are receptive to this, I may take some time to try to add in config options for placement as well in a follow up pull request. Just didn't want to waste any time learning that/trying it out if you weren't interested in this functionality. Appreciate the quick responses as well!

**Comment by mike-robertson** [comment]
@Justw8 fyi I am pretty sure there is some kind of defect with the base overkill code I built on top of. Got a few lua errors where it was trying to concat a null value in a string - I assume some kind of scenario where the overkill is null? I may have to do some checking on how to guard against that scenario

**Comment by mike-robertson** [comment]
Ok, I think I have a fix pushed up for that. I was able to do some testing and killing with an auto attack is what was causing the lua error. I tested after the fix and it seemed to display correctly. I also took the liberty of slightly refactoring the color code you had already so I could just reuse that.

You are going to want to double check my work here since I'm basically googling this + using existing code to understand the syntax & hoping lua linter doesn't let me down.

**Comment by Justw8** [comment]
Testing this out on the 9.1 PTR.

**Comment by Justw8** [comment]
This is in the alpha now.

**Pull request closed without merging**